Puc/v4p2/Plugin/UpdateChecker.php

```diff
@@ -399,6 +400,49 @@ public function addCheckForUpdatesLink($pluginMeta, $pluginFile) {
 		}
 
 		/**
+		 * Add a "View Details" link to the plugin row in the "Plugins" page. By default,
+		 * the new link will appear after the "Visit plugin site" link.
```

Comment:
Actually, I think this link would look better if it was *before* the "Visit plugin site" link. It's about consistency. For a plugin from wordpress.org, meta entries are in this order: 
- version
- author link
- view details
- everything else

Ideally, we should keep the same general order:
- version
- author link
- view details
- visit plugin site
- everything else

Maybe you could search $pluginMeta for a link to $pluginHeader['PluginURI'] (if set) and insert "View details" before that?